models.py

```diff
@@ -147,6 +147,61 @@ def generate_auth_token(self, expiration=600):
     @staticmethod
     def verify_auth_token(token):
         pass
+def find_by_username(username):
+    query = """SELECT username,password FROM tb_users WHERE username=(%s)"""
+    
+    conn = None
+    result = None
+    try:
+        params = dbconfig(filename, section)
```

Comment:
since you already have the connect method on line 22, you could make your logic a little more DRY by using the method to do the connection for you then you can do all the other queries and afterward close the connection.